draft-ietf-keytrans-architecture.md

```diff
@@ -211,7 +211,7 @@ potentially several encryption or signature public keys, and are meant to be
 verified with no/minimal network requests by the receiving users.
 
 In particular, credentials that can be verified with minimal network access are
-often required by applications provide anonymous communication. These
+often required by applications that provide anonymous communication. These
```

Comment:
maybe like this to avoid using `that + verb` twice in the same sentence?

```suggestion
often required by applications providing anonymous communication. These
```